runtime/wasm/src/host_exports.rs

```diff
@@ -41,6 +41,8 @@ pub(crate) struct HostExports {
     subgraph_id: SubgraphDeploymentId,
     pub(crate) api_version: Version,
     data_source_name: String,
+    data_source_address: Option<Address>,
+    data_source_network: String,
```

Comment:
Given that we'll be adding non-Ethereum data sources eventually, shouldn't this be `Option<String>`, like the address?